CONTRIBUTING.rst

```diff
@@ -99,19 +99,19 @@ to ask clarifying questions on the ticket as needed, too, if anything is unclear
 Step 1: Sign a Contribution Agreement
 =====================================
 
-Before edX can accept any code contributions from you, you'll need to sign
-the `individual contributor agreement`_ and send it in. This confirms
-that you have the authority to contribute the code in the pull request and
-ensures that edX can relicense it.
+Before edX can accept any code contributions from you, you'll need to sign the
+`individual contributor agreement`_. This confirms that you have the authority
+to contribute the code in the pull request and ensures that edX can re-license
+it.
+
+.. _individual contributor agreement: https://openedx.org/cla
 
-You should print out the agreement and sign it. Then scan (or photograph) the
-signed agreement and email it to the email address indicated on the agreement.
-Alternatively, you're also free to physically mail the agreement to the street
-address on the agreement. Once we have your agreement in hand, we can begin
-reviewing and merging your work.
+Once we have your signed agreement, we can begin reviewing and merging your
```

Comment:
Actually, let's rephrase since we are here to "Once we have received and processed your agreement, we will reach out to you by email to confirm. (Please make sure to indicate your email correctly in the agreement.) After that we will begin reviewing and merging your work.

Reply:
done